site/docs/latest/getting-started/docker.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Start a standalone cluster with Docker
+lead: Quickly start a Pulsar service in a single Docker container
+tags:
+- standalone
+- local
+- docker
+next: ../Clients
+---
+
+
+For the purposes of local development and testing, you can run Pulsar in {% popover standalone %}
+mode on your own machine within a Docker container.
+
+If you don't have Docker installed, you can download the [Community edition](https://www.docker.com/community-edition)
+and follow the instructions for your OS.
+
+## Starting Pulsar inside Docker
+
+```shell
+$ docker run -it \
+    -p 6650:6650 \
+    -p 8080:8080 \
+    -v $PWD/data:/pulsar/data \
+    apachepulsar/pulsar:{{site.current_version}} \
```

Comment:
Differently from the kubernetes yaml definitiions, this is part of the website, so it's getting converted into html at the website build time. At that point, Jekyll will substitute the variables with they're values.